`merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing`. It's because the file is not available in sandbox. You may disable these tests on darwin (if you believe it should work on darwin)

```
beads> --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention (0.85s)
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/shell_metacharacter_semicolon (0.12s)
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/shell_metacharacter_ampersand (0.11s)
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/shell_variable_expansion (0.11s)
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/command_substitution_backticks (0.11s)
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/command_substitution_dollar-paren (0.11s)
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/normal_backup_file (0.11s)
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/backup_with_spaces (0.11s)
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
beads>     --- FAIL: TestCleanupMergeArtifacts_CommandInjectionPrevention/path_traversal_attempt (0.06s)
beads>         merge_security_test.go:105: File "../../backup_etc_passwd" still exists after cleanup - this is OK if not tracked
beads>         merge_security_test.go:117: Command injection may have occurred - /etc/passwd missing
```

--- a/pkgs/by-name/be/beads/package.nix
+++ b/pkgs/by-name/be/beads/package.nix
@@ -1,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  gitMinimal,
+  installShellFiles,
+  versionCheckHook,
+  writableTmpDirAsHomeHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "beads";
+  version = "0.27.2";
+
+  src = fetchFromGitHub {
+    owner = "steveyegge";
+    repo = "beads";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-PpuyQCQocmOqt4EYDsjx1nh0dRxt2e7Vu1/KQ74B88Q=";
+  };
+
+  vendorHash = "sha256-5p4bHTBB6X30FosIn6rkMDJoap8tOvB7bLmVKsy09D8=";
+
+  subPackages = [ "cmd/bd" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  nativeCheckInputs = [
+    gitMinimal
+    writableTmpDirAsHomeHook
+  ];
+
+  # Skip security tests on Darwin - they check for /etc/passwd which isn't available in sandbox
+  checkFlags = lib.optionals stdenv.hostPlatform.isDarwin [
+    "-skip=TestCleanupMergeArtifacts_CommandInjectionPrevention"
+  ];
+
+  preCheck = ''
+    export PATH="$out/bin:$PATH"
+  '';
+
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    installShellCompletion --cmd bd \
+      --bash <($out/bin/bd completion bash) \
+      --fish <($out/bin/bd completion fish) \
+      --zsh <($out/bin/bd completion zsh)
+  '';
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+    writableTmpDirAsHomeHook
+  ];
+  versionCheckProgramArg = "version";
+  doInstallCheck = true;
+
+  meta = {
+    description = "Lightweight memory system for AI coding agents with graph-based issue tracking";
+    homepage = "https://github.com/steveyegge/beads";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ kedry ];
+    mainProgram = "bd";
+  };
+})

